pkg/oc/admin/prune/images.go

```diff
@@ -48,7 +51,9 @@ var (
 		registry by supplying --all=false flag.
 
 		By default, the prune operation performs a dry run making no changes to internal registry. A
-		--confirm flag is needed for changes to be effective.
+		--confirm flag is needed for changes to be effective. The flag requires a valid route to the
+		integrated Docker registry. If this command is run outside of the cluster, the route needs
+		to be provided using --registry-url.
```

Comment:
Sorry, but what does "run outside of the cluster" mean?

Reply:
**Inside** of the cluster means: from one of the nodes belonging to the OpenShift cluster with DNS managed either by master or by some internal DNS server.

Outside means: from anywhere else - and probably with NAT in the middle. 

Is there any less-confusing short description I could use?

Reply:
Oh, so `oc` should be able to reach the integrated registry either by internal IP or by external address from --registry-url. I see.

Reply:
would "outside of the cluster network" make this clearer, perhaps?

Reply:
> would "outside of the cluster network" make this clearer, perhaps?

done